src/server/server_family.cc

```diff
@@ -872,6 +884,30 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
                       &db_key_expire_metrics);
   }
 
+  // Command stats
+  {
+    const auto& state = *ServerState::tlocal();
```

Comment:
You read here the data from `ServerState::tlocal`. Don't you have to sum the data from all the thread pool?

Reply:
same for the resetting command

Reply:
Yes, you are correct! I'll move access to `ServerState` to the `GetMetrics` function.